.github/workflows/build.yml

```diff
@@ -4,18 +4,62 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-name: "wallet ci"
+name: "Wallet CI"
 
 env:
   GO_VERSION: 1.16
 
 on:
   push:
-    branches: [ main ]
+    branches: [main]
   pull_request:
-    branches: [ main ]
+    branches: [main]
 
 jobs:
+  Prettier:
+    name: Prettier Check
+    runs-on: ubuntu-20.04
+    timeout-minutes: 30
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.GO_VERSION }}
+      - name: Run Wallet Web Prettier Check
+        run: |
+          echo '127.0.0.1 testnet.orb.local' | sudo tee -a /etc/hosts
```

Comment:
Are you sure we need these hostfile entries?

Reply:
Not sure, should I try without them?

Reply:
We don;t need the host file - its only needed when we start docker and run

Reply:
yes, those are used for bddtests to avoid connection errors between containers

Reply:
Ok, thanks, removed